src/theme/extras/_modals.scss

```diff
@@ -34,4 +34,17 @@
   .chip {
     cursor: pointer;
   }
+
+  .overlay.loading-results {
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    z-index: 9999;
+    top: 0;
+    left: 0;
+    background-color: hsl(0deg 0% 100% / 64%);
```

Comment:
Hai provato la resa su Safari?
Te lo chiedo perche' di recente ho sbattuto il naso su delle noie di Safari con colori in hsl, e non supporta tutti i formati validi da specifiche css. In alcuni casi e con processors diversi (postcss e tailwind assicurato il fail) non usa i colori hsl se non sono scritti in formato legacy comma separated.

Esempio:
```
  --avatar: 34.29deg 77.78% 91.18%; -> rotto solo su Safari
  --avatar: 34.29deg, 77.78%, 91.18%; -> valido anche per Safari

```
Poi magari per quello che e' lo stack tecnologico qui non si rompe, ma un check rapido lo farei.

Reply:
si, funziona bene